ClientSide(FrontEnd)/Consumer_Side/src/components/Displayproduct.jsx

```diff
@@ -1,15 +1,15 @@
 import { useEffect , useState} from "react";
```

Comment:
Try to configure eslint to the repo and use recommended eslint plugins  / presets